src/gpio/hal_02.rs

```diff
@@ -39,16 +37,6 @@ impl<const P: char, const N: u8, MODE> StatefulOutputPin for Pin<P, N, Output<MO
     }
 }
 
-impl<const P: char, const N: u8, MODE> ToggleableOutputPin for Pin<P, N, Output<MODE>> {
```

Comment:
why are you removing this for e-h 0.2? i don't think that you should remove these traits yet (all 0.2 traits should be removed together at some point)

Reply:
thanks for catch. Fixed